design-system-docs/frontend/styles/_code.scss

```diff
@@ -18,8 +17,23 @@ pre code {
   overflow-x: auto;
   max-width: 100%;
   min-width: 100px;
-  padding: 20px;
   line-height: 1.5rem;
-  border: $cads-border-width-small $cads-language__border-colour solid;
   background-color: $cads-palette__white;
 }
+
+.highlight {
+  border: $cads-border-width-small $cads-language__border-colour solid;
+  padding: $cads-spacing-4;
+
+  &--with-button {
+    margin: 0;
+  }
+
+  button {
+    font-family: $cads-font-family;
+  }
+  > .highlight {
```

Comment:
I can't work out why, but Rouge and/or Kramdown is creating the following structure when converting a ``` code block in markdown into a `code` element:
`div.highlight > pre.highlight > code`

Not sure where the additional wrapping `div` is coming from, so I've had to add this into the SCSS